bindata/v4.1.0/config/defaultconfig.yaml

```diff
@@ -22,7 +22,7 @@ apiServerArguments:
   enable-aggregator-routing:
   - "true"
   shutdown-delay-duration:
-  - 70s # give SDN some time to converge: 30s for iptable lock contention, 25s for the second try and some seconds for AWS to update ELBs
+  - 30s # give SDN some time to converge: 30s for iptable lock contention, 25s for the second try and some seconds for AWS to update ELBs
```

Comment:
@sttts does this still apply `give SDN some time to converge: 30s for iptable lock contention, 25s for the second try`?

gcp-route-controller takes around 50 seconds (`FailureThreshold=10` * `Interval=5`) before it stops the gcp-route service. I still don't know what it achieves, but want to make sure our change has no adverse effect.
https://github.com/openshift/machine-config-operator/blob/master/cmd/gcp-routes-controller/run.go#L87-L103

and the gcp-service processes routes every 30s - https://github.com/openshift/installer/pull/3067/files#diff-f3a509446e9615909e1407b8d19b3dcdR81. I think the bash script has moved, I saw a PR that converts the script to go lang program but can't remember where it is.

Reply:
@squeed once told me about 30s retry duration for iptables.